src/app/find-help/reply-offer/reply-offer.component.html

```diff
@@ -60,7 +60,7 @@ <h3>{{ "REPLY_OFFER" | translate }}</h3>
           </mat-error>
         </mat-form-field>
       </div>
-      <div class="row mt-3 mb-3">
+      <div *ngIf="!authService.isLoggedIn()" class="row mt-3 mb-3">
```

Comment:
Is logged in could be invoked in a component. Here it calls this funcion up to 300 per second.